Packages/com.unity.inputsystem/InputSystem/Editor/AssetEditor/ParameterListView.cs

```diff
@@ -278,11 +278,26 @@ void OnEditEnd()
 
                 if (parameter.isEnum)
                 {
-                    var intValue = parameter.value.value.ToInt32();
-                    var field = new DropdownField(label.text, parameter.enumNames.Select(x => x.text).ToList(), intValue);
-                    field.tooltip = label.tooltip;
-                    field.RegisterValueChangedCallback(evt => OnValueChanged(ref parameter, field.index, closedIndex));
-                    field.RegisterCallback<BlurEvent>(_ => OnEditEnd());
+                    var names = parameter.enumNames.Select(c => c.text).ToList();
+                    var rawValue = parameter.value.value.ToInt32();
```

Comment:
Minor: It seems the only place this rawValue is used is on the following line so why not pass it directly?